src/gtk-3.0/gtk-widgets.css

```diff
@@ -822,7 +852,7 @@ dialog filechooser .path-bar image {
 	margin-right: 5px;
 }
 dialog filechooser .frame.sidebar separator {
-	border-image: url("assets/horizontal-line.png") 1 / 1px stretch;
+	border-image: url("assets/horizontal-line.svg") 1 / 1px stretch;
```

Comment:
Vertical lines are blurry

Reply:
Or maybe the screenshot itself is blurry?

Reply:
Dunno. looks ok to me in mtpaint and in the browser.

Reply:
@zigbert uses 100x100 in svgs. I used 192x192 and check mark looks better.
![l3](https://user-images.githubusercontent.com/1019119/187130788-b5fb41e4-4ea5-4e24-9d8b-6e4e566ad401.png)